Cleanup Hosting options for an MCP world (#471)
ArcadeAI/docs: app/en/home/hosting-overview/page.mdx

# Hosting Options

> The easiest and best way to use Arcade is to use our cloud service - sign up for free at [https://api.arcade.dev](https://api.arcade.dev).  However, you might need to connect your tools to local resources (e.g. a local database or filesystem) or keep data within your own infrastructure.  Don't worry, Arcade has you covered!

Reviewer: ```suggestion
The easiest and best way to use Arcade.dev is via our cloud service - sign up for free at [https://api.arcade.dev](https://api.arcade.dev).

In some cases, you might need to connect your tools to local resources (e.g. a local database or filesystem), or keep data within your own infrastructure.  Don't worry, Arcade has you covered!
```

## MCP Server Deployment

> You can route and manage tool calls from your agents to MCP servers hosted anywhere - on your machine, on your own infrastructure, in a private cloud, or on our cloud service.  This allows you to mix the best public tools with your own private tools.

Reviewer: ```suggestion
You can route and manage tool calls from your agents to MCP servers hosted anywhere: on your machine, on your own infrastructure, in a private cloud, or on our cloud service.  This allows you to mix the best public tools with your own private tools.
```

> See [On-premise MCP Servers](/home/deployment/on-prem-mcp) for more information about how to use your own MCP servers in a federated deployment, and see [Arcade Deploy](/home/serve-tools/arcade-deploy) to learn how to deploy to Arcade's cloud.

Reviewer: ```suggestion
See [On-premise MCP Servers](/home/deployment/on-prem-mcp) for more information about how to use your own MCP servers running anywhere, and see [Arcade Deploy](/home/serve-tools/arcade-deploy) to learn how to deploy to Arcade's cloud.
```

## Customizing Auth

> You don't have to host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, but you can always provide your own OAuth app credentials. We recommend doing this for any production use-case so that you can have isolated rate limits with the OAuth service provider and give your users a consistent experience when they go through an auth flow.

Reviewer: ```suggestion
You don't have to self-host Arcade to customize your auth experiences. Arcade's cloud service supports a number of auth providers out of the box, and you can provide your own OAuth app credentials to brand your end-user experience. We recommend doing this for all production use cases, so that you can have isolated rate limits with the OAuth service provider and you can give your users a consistent experience when they go through an auth flow.
```
